srcpkgs/mc/patches/xbps.patch

```diff
@@ -7,8 +7,8 @@
 -# .tar.xz, .txz
 +# .tar.xz, .txz, .xbps
  [tar.xz]
--Regex=\.t(ar\.xz|xz)$
-+Regex=\.(t(ar\.xz|xz)|xbps)$
+-Regex=\\.t(ar\\.xz|xz)$
++Regex=\\.(t(ar\\.xz|xz)|xbps)$
```

Comment:
As stated above, there are no correct places for it, it could be anywhere
This mechanism does autodetection of compression, and seemingly can't even be forced to read an archive differently than what the autodetection picks.